xml/admin_gui_dashboard.xml

```diff
@@ -14,23 +14,24 @@
   </dm:docmanager>
  </info>
  <para>
-  The &dashboard; is a module that adds a built-in Web based monitoring and
-  administration application to the &mgr; (refer to
-  <xref linkend="storage-intro-core-nodes" /> for more details on &mgr;). You
-  no longer need to know complex &ceph; related commands to manage and monitor
-  your &ceph; cluster. You can either use the &dashboard;'s intuitive Web
-  interface, or its built-in REST API.
+   The &dashboard; is a built-in web-based &ceph; management and monitoring application
+   that administers various aspects and objects of the cluster. The dashboard is
+   automatically enabled after the basic cluster is deployed
+   in <xref linkend="deploy-cephadm-day1"/>.
  </para>
  <para>
-  The &dashboard; is automatically enabled and configured with &deepsea;'s
-  stage 3 during the deployment procedure (see DEAD LINK). In a &ceph; cluster
-  with multiple &mgr; instances, only the dashboard running on the currently
-  active &mgr; daemon will serve incoming requests. Accessing the dashboard's
-  TCP port on any of the other &mgr; instances that are currently on standby
-  will perform an HTTP redirect (303) to the currently active &mgr;'s dashboard
-  URL. This way, you can point your browser to any of the &mgr; instances in
-  order to access the dashboard. Consider this behavior when securing access
-  with firewall or planning for HA setup.
+   The &dashboard; for &productname; &productnumber; has added more web-based
+   management capabilities to make it easier to administer &ceph;, including
+   web based monitoring and administration application to the &mgr;.
```

Comment:
That sentence sounds odd to me: "including web based administration application"?

Reply:
That should be "application administration to the &mgr;"  - good catch